patch-package
--- a/node_modules/system-components/dist/System.js
+++ b/node_modules/system-components/dist/System.js
@@ -106,6 +106,7 @@ var System = function System(opts) {
       return _react2.default.createElement(_cleanTag2.default, props);
     };
     div.defaultProps = { blacklist: blacklist };
+    div.styledComponentId = 'lol';
 
     var Component = createComponent(div).apply(undefined, _toConsumableArray(funcs));
 
